contracts/6/forwarder/ERC20ForwardRequestCompatible.sol

```diff
@@ -0,0 +1,18 @@
+pragma solidity ^0.6.8;
+pragma experimental ABIEncoderV2;
+
+contract ERC20ForwardRequestCompatible{
```

Comment:
Rename to ERC20ForwardRequestTypes

"Compatible" is not appropriate for name for this contract.